region_migrations/watch.go

```diff
@@ -0,0 +1,80 @@
+package region_migrations
+
+import (
+	"fmt"
+	"net/url"
+
+	"github.com/Scalingo/cli/config"
+	scalingo "github.com/Scalingo/go-scalingo"
+	"github.com/fatih/color"
+	errgo "gopkg.in/errgo.v1"
+)
+
+const (
+	SUCCESS = "✔"
+	ERROR   = "✘"
+)
+
+func WatchMigration(client *scalingo.Client, appId, id string) error {
+	refresher := NewRefresher(client, appId, id)
+	migration, err := refresher.Start()
+	if err != nil {
+		return errgo.Notef(err, "fail to watch migration")
+	}
+
+	if migration == nil {
+		return nil
+	}
+
+	if migration.Status == scalingo.RegionMigrationStatusDone {
+		newRegionClient, err := config.ScalingoClientForRegion(migration.Destination)
+		if err != nil {
+			color.Green("The application has been migrated!\n")
+			return nil
+		}
+
+		app, err := newRegionClient.AppsShow(appId)
+		if err != nil {
+			color.Green("The application has been migrated!\n")
+			return nil
+		}
+
+		domains, err := newRegionClient.DomainsList(appId)
+		if err != nil {
+			color.Green("The application has been migrated!\n")
+			return nil
+		}
+
+		color.Green("Your application is now available at: %s\n\n", app.BaseURL)
+		if len(domains) > 0 {
+			parsed, err := url.Parse(app.BaseURL)
+			if err != nil {
+				fmt.Printf("You need to change the DNS record of your domains to point to the new region.")
```

Comment:
I would love the message to be super informative like: `set the CNAME to my-app.scalingo.io` in case it's not a root domain and `set the A record to XX.XX.XX.XX` in case of root domain. Would it be possible?

Reply:
https://www.youtube.com/watch?v=jv9sDn_2XkI

Reply:
Seriously, i'll check if it's possible.

Reply:
Blou